bouncer/commands/new_swap.ts

```diff
@@ -77,8 +82,11 @@ async function newSwapCommand() {
     args.refundAddress !== undefined && args.minPrice !== undefined
       ? {
           retryDurationBlocks: args.refundDuration,
-          refundAddress: args.refundAddress,
-          minPrice: args.minPrice,
+          refundAddress:
+            args.sourceAsset === 'Dot'
+              ? decodeDotAddressForContract(args.refundAddress)
+              : args.refundAddress,
+          minPrice: usdcToInternalPrice(args.sourceAsset as InternalAsset, args.minPrice),
```

Comment:
the example has Dot -> Btc, how is the price represented in that case?

Reply:
The price you would enter is the min price of DOT in USDC. say 4, because DOT is worth $4.15 right now.

Reply:
but what about the price of bitcoin?

Reply:
oh sorry. I misunderstood what price is. A little confusing 😅.

Reply:
Ok i think i got it correct now. The "price" is in Dot per Btc and then converts to dot_planck/Satoshi taking into account the decimals of both assets. This internally works out to a minimum output in Satoshi. I checked the minimum output to confirm it was what i expected.